src/Sylius/Bundle/CoreBundle/PaymentRequest/CommandHandler/Payum/TokenPaymentRequestHandler.php

```diff
@@ -44,7 +44,9 @@ public function __invoke(PaymentRequestHashAwareInterface $command): void
         $request = $this->payumRequestFactory->createNewWithToken($token);
 
         $token = $request->getToken();
-        Assert::notNull($token, 'A Payum token cannot be null.');
+        if (null === $token) {
+            throw new NullPaymentTokenException();
```

Comment:
Exception name should indicate that this is payum token exception and not sylius payment token exception.